From 303b3ab0a81270f7bbd258086a8b61e2cefb5dbb Mon Sep 17 00:00:00 2001
From: Ilya Maximets <i.maximets@ovn.org>
Date: Fri, 13 Dec 2019 13:52:10 +0100
Subject: cirrus: Use FreeBSD 12.1 stable release.

freebsd-12-0-snap image family suddenly removed from the gCloud,
so can not be used anymore.  Updating to more recent 12.1 releases.

Signed-off-by: Ilya Maximets <i.maximets@ovn.org>
Acked-by: Ben Pfaff <blp@ovn.org>
---
 .cirrus.yml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/.cirrus.yml b/.cirrus.yml
index aaa5e3eeb..48ad3e3c2 100644
--- a/.cirrus.yml
+++ b/.cirrus.yml
@@ -2,7 +2,7 @@ freebsd_build_task:
 
   freebsd_instance:
     matrix:
-      image_family: freebsd-12-0-snap
+      image_family: freebsd-12-1-snap
       image_family: freebsd-11-3-snap
     cpu: 4
     memory: 8G
-- 
2.14.1


